utils/database.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+import asyncpg
+
+async def create_pool():
+    pool = await asyncpg.create_pool(dsn='postgresql://guildbot:gu1ldb0t@localhost/guildbot')
```

Comment:
Move db info out of version control
```suggestion
    pool = await asyncpg.create_pool(user=config.db_user, password=config.db_pass, host=config.db_host, database=config.db_name)
```